integration_tests/test_suites/daemon-test-suite/monitoring_daemon_tests/test_monitoring.py

```diff
@@ -71,6 +72,8 @@ def test_monitoring():
             assert all_daemons_healthy(instance)
 
 
+@pytest.skip("Disabling due to timeouts")
+# https://github.com/dagster-io/dagster/issues/8905
```

Comment:
I think its worth trying to bump the timeout instead, the test fails relatively fast so i think its fine to give it some more time